dll/opengl/opengl32/glapi_x86.s

```diff
@@ -7,19 +7,47 @@
 
 /* X86 opengl API entry points, fast forward to the current thread's dispatch table */
 #include <asm.inc>
+#include <ks386.inc>
+
+.data
+ASSUME nothing
+
+.align 16
 
 .code
 
+#ifdef OPENG32_USE_TLS
 EXTERN _OglTlsIndex:DWORD
 EXTERN _TlsGetValue@4:PROC
+#endif
 
 MACRO(USE_GL_FUNC, name, offset, stack)
 PUBLIC _gl&name&@&stack
 .PROC _gl&name&@&stack
+    LOCAL fast_ret
+
+#ifdef OPENG32_USE_TLS
```

Comment:
Is there a reason why we keep these code paths around?
Do we want 2 opengl32 dll's next to eachother, so that we can test both paths? (opengl32.dll and opengl32_teb.dll or so)

Reply:
@learn-more I consider removing the code path for TLS altogether, yes